### PR TITLE
Bz1972934 - add MTU requirements info

### PR DESCRIPTION
**Pull request by dcdacosta**
[Bug fix]

Fixes issue https://bugzilla.redhat.com/show_bug.cgi?id=1972934

Changes proposed in this pull request:

- Added and MTU Requirements topic to the Networking Requirements chapter.

Direct preview link: https://ovirt.github.io/ovirt-site/previews/2892/documentation/installing_ovirt_as_a_standalone_manager_with_local_databases/index.html#MTU_Requirements_SM_localDB_deploy

I confirm that this pull request was submitted according to the [contribution guidelines](https://github.com/oVirt/ovirt-site/blob/master/CONTRIBUTING.md): (@dcdacosta )

This pull request needs review by: (Marina Kalinin)
This pull request needs peer review by: (@emarcusRH )

### CODE REVIEW

**Review by almusil — APPROVED** [review]
LGTM, thanks

**Comment by mburman5** [comment]
LGTM, thanks

**Review by sandrobonazzola — APPROVED** [review]

File: source/documentation/common/prereqs/ref-MTU_Requirements.adoc [hunk 1]
```diff
@@ -0,0 +1,9 @@
+:_content-type: CONCEPT
+[id='MTU_Requirements_{context}']
+= Maximum Transmission Unit Requirements
+
+// Included in:
+// PPG
+// Install
+
+The recommended Maximum Transmission Units (MTU) setting for Hosts during deployment is 1500. It is possible to update this setting after the environment is setup to a different MTU. For more information on changing the MTU setting, see link:https://access.redhat.com/solutions/4129641[How to change the Hosted Engine VM network MTU].
```
Comment (emarcusRH) [inline source/documentation/common/prereqs/ref-MTU_Requirements.adoc:9]: "the environment is set up" - two words for set up as an action (setup, one word as a noun)

**Review by emarcusRH — APPROVED** [review]
one correction, 
/lgtm

File: source/documentation/common/prereqs/ref-MTU_Requirements.adoc (hunk 1 above)
Comment (dcdacosta) [reply source/documentation/common/prereqs/ref-MTU_Requirements.adoc]: Fixed.

**Pull request merged**